airflow-core/src/airflow/ui/src/pages/Connections/Connections.tsx

```diff
@@ -58,6 +60,7 @@ const columns: Array<ColumnDef<ConnectionResponse>> = [
     cell: ({ row: { original } }) => (
       <Flex justifyContent="end">
         <DeleteConnectionButton connectionId={original.connection_id} disabled={false} />
+        <EditConnectionButton connection={original} disabled={false} />
```

Comment:
In the variables form the edit is in front of the delete - which is better in my view.